packages/alchemy/src/config/utils/cookies.ts

```diff
@@ -0,0 +1,83 @@
+import { getCookie, removeCookie, setCookie } from "typescript-cookie";
+import { DEFAULT_SESSION_MS } from "../../signer/session/manager.js";
+import type { AlchemyAccountsConfig, AlchemyClientState } from "../types";
+import { deserialize } from "./deserialize.js";
+
+/**
+ * Function to create cookie based Storage
+ *
+ * @param config optional config object that allows you to set the session length
+ * @returns an instance of a browser storage object that leverages cookies
+ */
+export const cookieStorage: (config?: { sessionLength: number }) => Storage = (
+  config
+) => ({
+  // this is unused for now, we should update this if we do need it
+  length: 0,
+
+  clear: function (): void {
+    if (typeof document === "undefined") return;
+
+    document.cookie = "";
```

Comment:
This feels slightly sketchy. The app using aa-sdk may have its own cookies that it uses for storage unrelated to aa-sdk and wouldn't expect aa-sdk to delete its cookies under any circumstances.

I'm not sure if this is actually called anywhere from the SDK. If not, I might consider removing it from this interface or making it throw an error to stop future code from using it carelessly.

I get that local/session storage has this same issue so I don't think this is necessarily that bad, just wanted to bring it up.

Reply:
yea that's a good call, I was back and forth on this as well. It's a generic cookie storage interface that matches the browser `Storage` so it felt ok to do, but it's within the context of aa-sdk so I think you're right that as a dev I'd be annoyed if it cleared my cookies. 

It doesn't get used anywhere in the sdk itself, so I'll just throw here